### PR TITLE
TC-BINFO-1.1 yaml

### PR DESCRIPTION
**Pull request by cecille**
#### Problem
BINFO 1.1 didn't have runnable yaml

#### Change overview
yaml-izes the test

#### Testing
chip-tool and all clusters app

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml [hunk 1]
```diff
@@ -23,87 +23,70 @@ config:
...
+                      1,
+                      2,
+                      3,
+                      4,
+                      5,
+                      6,
+                      7,
+                      8,
+                      9,
+                      0xA,
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml:73]: Looks like a more careful version of https://github.com/project-chip/connectedhomeip/pull/21289/files ?

Needs to regenerate darwin-framework-tool...

**Review by bzbarsky-apple — APPROVED** [review]
To actually automate this, doesn't it need to move out of the manual tests into automated ones in src/app/tests/suites/tests.js?

File: src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml (hunk 1 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml:73]: ah, yeah - we've been attempting to get some of these things automated so you'll see a few of these. Let me close this one.

**Comment by cecille** [comment]
Martin has the same thing up - let's let that one go in so we have a few more people go through the review process for these tests.

**Pull request closed without merging**